server/src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -428,6 +439,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.field("state_uuid", stateUUID);
             builder.field("last_committed_config", lastCommittedConfiguration);
             builder.field("last_accepted_config", lastAcceptedConfiguration);
+            // TODO include voting tombstones here
```

Comment:
why not do it?

Reply:
We need to discuss how it should be represented in JSON and I did not want that discussion to block this PR.

Reply:
ok, I hope we won't forget about this :) Can you add a note to the meta issue?

Reply:
Yes, I added a point for that.